I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/code/biome-generator-screen.rpy", line 2, in script
    $ renpy.pause()
  File "game/code/biome-generator-screen.rpy", line 2, in <module>
    $ renpy.pause()
  File "game/code/biome-generator-screen.rpy", line 15, in execute
    screen biome_generator():
  File "game/code/biome-generator-screen.rpy", line 15, in execute
    screen biome_generator():
  File "game/code/biome-generator-screen.rpy", line 20, in execute
    hbox:
  File "game/code/biome-generator-screen.rpy", line 22, in execute
    imagebutton auto "gui/button/Terrestrial_Biome_Placeholder_Button_%s.png" xpos 640 ypos 660 focus_mask True action MainMenu()
Exception: Imagebutton does not have a idle image. (auto=u'gui/button/Terrestrial_Biome_Placeholder_Button_%s.png').

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/code/biome-generator-screen.rpy", line 2, in script
    $ renpy.pause()
  File "renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/code/biome-generator-screen.rpy", line 2, in <module>
    $ renpy.pause()
  File "renpy/exports.py", line 1563, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay)
  File "renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "renpy/display/core.py", line 3325, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "renpy/display/core.py", line 3737, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/screen.py", line 436, in visit_all
    callback(self)
  File "renpy/display/core.py", line 3737, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/screen.py", line 447, in per_interact
    self.update()
  File "renpy/display/screen.py", line 637, in update
    self.screen.function(**self.scope)
  File "game/code/biome-generator-screen.rpy", line 15, in execute
    screen biome_generator():
  File "game/code/biome-generator-screen.rpy", line 15, in execute
    screen biome_generator():
  File "game/code/biome-generator-screen.rpy", line 20, in execute
    hbox:
  File "game/code/biome-generator-screen.rpy", line 22, in execute
    imagebutton auto "gui/button/Terrestrial_Biome_Placeholder_Button_%s.png" xpos 640 ypos 660 focus_mask True action MainMenu()
  File "renpy/ui.py", line 951, in _imagebutton
    idle = choice(idle, idle_image, "idle", required=True)
  File "renpy/ui.py", line 945, in choice
    raise Exception("Imagebutton does not have a %s image. (auto=%r)." % (name, auto))
Exception: Imagebutton does not have a idle image. (auto=u'gui/button/Terrestrial_Biome_Placeholder_Button_%s.png').

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Biome Generator 1.0
Sun Apr 10 15:40:11 2022
